docs/executor.rst

```diff
@@ -38,9 +38,9 @@ you have access.
 .. note::
     A friendly FYI: the tutorial endpoint is public -- available for any
     (authenticated) user.  You are welcome to use it, but please limit the size
-    and number of functions you send to this endpoint as it is a shared
-    resource that is (intentionally) not very powerful.  It's primary intended
-    purpose is for an introduction to the Globus Compute toolset.
+    and number of functions you send to this endpoint.  It is hosted on a small
+    VM with limited CPU and memory, intentionally underpowered.   Its primary
+    intended purpose is for an introduction to the Globus Compute toolset.
```

Comment:
It would be good to mention here as well as in [tutorial.rst](https://github.com/funcx-faas/funcX/blob/a7efc9096f7a5939dced4bd405a662a4da80f82e/docs/tutorial.rst?plain=1#L41-L45) that users can not create public endpoints.  Perhaps something like
> This endpoint has been made public by the Globus Compute team for the purposes of this tutorial, but endpoints created by users can not be shared publicly.

Reply:
Added to both quickstart.rst and executor.rst

Reply:
Thanks, can we also update the tutorial.rst note I reference above?

Reply:
Ah, didn't see that it was a link, and thought you meant the corresponding quickstart one.

Added - Now there's 3 instances of the phrase, thought slightly different depending on section/grammar/word wraps.

Reply:
Thanks!